### ISSUE TITLE
multiple tables cell edit

### ISSUE THREAD

**Issue by texas697** [AllenFang/react-bootstrap-table#1446]
I am displaying multiple tables.
```
`  {newSections.map((section, index) => (
          <div key={index}>
            <br />
            <h5>{section[0].name}</h5>
            <div className='lot-recon-table'>
              <BootstrapTable condensed data={section} cellEdit={cellEditPropNew}>
                <TableHeaderColumn dataField='qtr' width='70' editColumnClassName='p-0' editable={false} isKey>Qtr</TableHeaderColumn>
                <TableHeaderColumn dataField='name' hidden />
                <TableHeaderColumn dataField='min' width='90' dataFormat={this._priceFormatter} editColumnClassName='p-0' editorClass='input-xs' editable={{ type: 'number' }}>Min Price</TableHeaderColumn>
                <TableHeaderColumn dataField='max' width='90' dataFormat={this._priceFormatter} editColumnClassName='p-0' editorClass='input-xs' editable={{ type: 'number' }}>Max Price</TableHeaderColumn>
                <TableHeaderColumn dataField='occ' width='70' editColumnClassName='p-0' editorClass='input-xs' editable={{ type: 'number' }}>Occ</TableHeaderColumn>
                <TableHeaderColumn dataField='mod' width='70' editColumnClassName='p-0' editorClass='input-xs' editable={{ type: 'number' }}>Mod</TableHeaderColumn>
                <TableHeaderColumn dataField='fv' width='70' editColumnClassName='p-0' editorClass='input-xs' editable={{ type: 'number' }}>Fin</TableHeaderColumn>
                <TableHeaderColumn dataField='uc' width='70' editColumnClassName='p-0' editorClass='input-xs' editable={{ type: 'number' }}>UC</TableHeaderColumn>
                <TableHeaderColumn dataField='vdl' width='70' editColumnClassName='p-0' editorClass='input-xs' editable={{ type: 'number' }}>VDL</TableHeaderColumn>
                <TableHeaderColumn dataField='fut' width='70' editColumnClassName='p-0' editorClass='input-xs' editable={{ type: 'number' }}>Fut</TableHeaderColumn>
                <TableHeaderColumn dataField='con' width='70' editColumnClassName='p-0' editorClass='input-xs' editable={{ type: 'number' }}>Conc</TableHeaderColumn>
                <TableHeaderColumn dataField='tot' width='70' editable={false}>Calc Total</TableHeaderColumn>
                <TableHeaderColumn dataField='storeTot' width='70' editable={false}>Total Lots</TableHeaderColumn>
                <TableHeaderColumn dataField='variance' width='70' editable={false}>Variance</TableHeaderColumn>
              </BootstrapTable>
            </div>
          </div>
        ))}`
```

When I change a input value the wrong table is being updated. The last table is always being updated. This is how I am trying to get it to work. 

```
`  _onAfterSaveCellNew = (row, cellName, cellValue) => {
    const { newSections } = this.props
    let newSectionsArry = _.each(newSections, item => {
      item.map(a => Object.assign({}, a))
      return item
    })

    newSectionsArry = _.filter(newSectionsArry, arry => {
      return _.filter(arry, item => {
        if (item.qtr === row.qtr && item.name === row.name && item.storeTot === row.storeTot) {
          // eslint-disable-next-line
          item[cellName] = parseInt(cellValue)
          item.tot = item.occ + item.mod + item.fv + item.uc + item.vdl + item.fut + item.con
        }
        return item
      })
    })
    this.props.onUpdateNewSections(newSectionsArry)
  }`
```
The row value is incorrect, it is defaulting to the last table in the array. Is there a ref I need to use somewhere?

**Comment by AllenFang** [comment]
@texas697 sorry for lately reply, is that same as https://github.com/AllenFang/react-bootstrap-table/issues/1428 ?, if yes, let's back to https://github.com/AllenFang/react-bootstrap-table/issues/1428 and close this issue, thanks

**Comment by AllenFang** [comment]
@texas697, this bug was fixed on `v3.4.6`, please let me know if the bug still remains, thanks 👍
